### PR TITLE
Categories

### PR DESCRIPTION
**Pull request by mirhamasala**
## Notes

~ Design isn't final - We still need to work on it with Filipa

- [x] Check log error
- [ ] Missing ecosystem subcategories (needs to be populated by content team)
- [ ] Missing validation of YAML files (tech debt)
- [x] Missing results and reset for mobile
- [x] Missing total category amount for each category
- [x] Show NoResultsMessage in place of the results

### CODE REVIEW

File: src/app/_components/ListboxButton.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+import { Listbox } from '@headlessui/react'
+
+type ListboxButtonProps = {
+  ariaLabel: string
+  open: boolean
+  children: React.ReactNode
+}
+
+export const ListboxButton: React.FC<ListboxButtonProps> = ({
```
Comment (mirhamasala) [inline src/app/_components/ListboxButton.tsx:11]: Use function declaration

**Comment by CharlyMartin** [comment]
**To check**

- [ ] When manually changing a filter param in the URL, the page doesn't update accordingly

**Review by mirhamasala — COMMENTED** [review]
Requires more attention @CharlyMartin >>

- Category.tsx
- cmsConstants.ts
- useCategory.ts
- useResetSearchParams.ts
- categoryTypes.ts
- categoryUtils.ts
- yamlUtils.ts

**Comment by mirhamasala** [comment]
Thank you, @CharlyMartin ❤️

**Pull request merged**